Exercise_E.js

```diff
@@ -34,7 +34,12 @@ var KEYS = {
 };
 
 function maya(number) {
-  // CODE HERE
+let sentence = number.toString()
+let answer = "";
+for (let i = 0; i < sentence.length ; i++) {
+  answer = answer + KEYS[sentence[i]]
+}
+return answer
 }
 

```

Comment:
Great work, if you have time you should come back and see if your able to use less variable within this function.